examples/space_invaders/cartpole.py

```diff
@@ -0,0 +1,63 @@
+import pdb
```

Comment:
I don't think we want a cartpole experiment in the `space_invaders` directory. Can this be moved to a new directory named `cartpole`?

Reply:
Yeah it was mistake. I would add new directory when I'm ready.